.github/workflows/test.yaml

```diff
@@ -71,8 +80,7 @@ jobs:
 
       - name: Lint Rust
         run: |
-          # Currently the only check is clippy, could bundle the other checks into the flake and remove the steps above?
-          nix flake check --all-systems
```

Comment:
This all system check was downloading things for other systems and I don't think it's ever caught an actual problem. We test on linux and macos, that's covering everything we're actually using.